pkg/ccl/backupccl/restore_planning_test.go

```diff
@@ -151,24 +153,37 @@ func TestBackupManifestVersionCompatibility(t *testing.T) {
 			minimumSupportedVersion: roachpb.Version{Major: 22, Minor: 2},
 			expectedError:           "the backup is from a version older than our minimum restorable version 22.2",
 		},
+		{
+			name:                    "online-restore-old-backup",
+			backupVersion:           roachpb.Version{Major: 23, Minor: 2},
```

Comment:
@dt Sadly this unit test instead fails with `cannot upgrade to 24.1: node running 23.2`. Does this have something to do with minting 24.1?